### PR TITLE
Feat/login

### PR DESCRIPTION
**Pull request by raphael-arce**
this is WIP and the ticket https://app.asana.com/0/1204152181784036/1206727518884851 is not fully implemented yet but register/login works (I tested only locally so far though) and @Jaszkowic needs it to continue on other features. @aeschi and me will continue to implement the rest of this feature after the merge.

### CODE REVIEW

File: src/components/navbar/navbar.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const Navbar: React.FC = () => {
             className={`
             flex h-14 w-14 flex-col items-center justify-center rounded-xl pt-1 text-sm font-medium
             hover:bg-blue-600 hover:bg-opacity-10 lg:h-16 lg:w-16 lg:rounded lg:text-base
-            ${url.pathname === item.path ? "bg-blue-600 bg-opacity-10 text-blue-600" : "text-gray-800"}
+            ${url.pathname.split("/", 2).join("/") === item.path ? "bg-blue-600 bg-opacity-10 text-blue-600" : "text-gray-800"}
```
Comment (raphael-arce) [inline src/components/navbar/navbar.tsx:49]: let's maybe use `url.pathname.startsWith(item.path)` instead

File: src/components/profile/profile-logged-in.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import PasswordReset from "./password-reset";
+
+const ProfileLoggedIn: React.FC = () => {
+  const i18n = useI18nStore().i18n();
+  const { logout } = useAuthStore();
+  const { url } = useUrlState();
+  const authType = url.searchParams.get("mode");
+
+  if (authType === "reset-password") return <PasswordReset />;
+  else {
```
Comment (raphael-arce) [inline src/components/profile/profile-logged-in.tsx:18]: we don't need an else if we have a return in the if, I personally like this format:

```js
if (condition) {
  return ...
}

return ...
```

**Review by Jaszkowic — APPROVED** [review]
Auth flow is too complex to review this properly code-wise. As you did it pairing @aeschi  @raphael-arce , I think it's fine. I tested login, logout locally and it works fine. 

Still missing: i18n support

I think merging is okay and tackling everything else in follow-up PRs.

File: src/auth/supabase-client.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { createClient } from "@supabase/supabase-js";
+
+const SUPABASE_URL = import.meta.env.VITE_SUPABASE_URL
```
[the remark this answers is not in the document]
Comment (raphael-arce) [reply src/auth/supabase-client.ts:3]: valid point 👍

**Pull request merged**